mkdocs.yml

```diff
@@ -1,34 +1,35 @@
+
 site_name: Armbian Documentation
 site_author: "Armbian team"
 
-copyright: Copyright &copy; 2015 - 2024 Armbian team
+copyright: Copyright &copy; 2015 - 2022 Armbian team
```

Comment:
why?

Reply:
Why did this change? Perhaps someone manually edited the date instead or fixing the tool that generated this file.